docs: fixed typos and small enhancements in en docs, fixed dead links in de #567

colinscz opened the pull request:
<!-- PULL REQUEST TEMPLATE -->
<!-- (Update "[ ]" to "[x]" to check a box) -->

### 🔗 Linked issue
Closes #567 

<!-- Please ensure there is an open issue and mention its number as #123 -->

### ❓ Type of change

<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Fixed some typos in the documentation and adjusted the descriptions where I saw fit so the "reading flow" is a bit better :) 
I checked this draft PR and it seems that my changes do not overlap with it: https://github.com/Tresjs/tres/pull/495

Also there were some german links that the linter marked as dead or incorrect, I fixed those too.
Resolves #567 

### 📝 Checklist

<!-- Put an `x` in all the boxes that apply. -->
<!-- If your change requires a documentation PR, please link it appropriately -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] I have linked an issue or discussion.
- [x] It's submitted to the `main` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] I have updated the documentation accordingly.
- [x] All tests are passing
- [ ] New/updated tests are included

**Other information:**

colinscz commented on docs/guide/index.md:
```diff
@@ -44,7 +44,7 @@ pnpm add @types/three -D
 ## Vite
 
-If you are using Vite, you have add the following to your `vite.config.ts`:
+If you are using Vite, you just need to add this to your `vite.config.ts` inside of the vue plugin:
```
Here I am unsure, the wording on the installation and the index page is slightly different - maybe it makes sense to use the same wording on both pages?

colinscz replied:
I added the same sentence to both getting started and index page in english - hope that works for you guys

andretchen0 commented on docs/examples/lights-shadows.md:
```diff
@@ -62,11 +62,11 @@ import { OrbitControls } from '@tresjs/cientos'
 As you know every instance in [ThreeJs](https://threejs.org/) is available in **TresJs** so are all the light types, we just need to add the `Tres` prefix to use them.
 
-But not all lights can cast shadows, this definition comes directly from ThreeJs and makes sense, for example the purpose of an [ambientLight](https://threejs.org/docs/index.html?q=ambient#api/en/lights/AmbientLight) is to iluminate everysingle side of your scene, so it makes no sense for it to cast shadows, on the contrary, a [DirectionalLight](https://threejs.org/docs/index.html?q=light#api/en/ ...
+But not all lights can cast shadows, this definition comes directly from ThreeJs and makes sense, for example the purpose of an [ambientLight](https://threejs.org/docs/index.html?q=ambient#api/en/lights/AmbientLight) is to iluminate every single side of your scene, so it makes no sense for it to cast shadows, on the contrary, a [DirectionalLight](https://threejs.org/docs/index.html?q=light#api/en ...
```
(Non-blocking, but since we're at it ...)

Small typo here.

> iluminate

> illuminate

andretchen0 commented on docs/examples/load-models.md:
````diff
@@ -57,7 +57,7 @@ import { useGLTF } from '@tresjs/cientos'
 ```
 
-Alternatively you can easily select Objects inside the model using `nodes` property
+Alternatively you can easily select objects inside the model using `nodes` property.
````
(Non-blocking)

> the model using `nodes` property.

> the model using the `nodes` property.

andretchen0 commented on docs/examples/orbit-controls.md:
````diff
@@ -46,7 +46,7 @@ Now you can use the `TresOrbitControls` component in your scene.
 ```
 
-Since [OrbitControls](https://threejs.org/docs/index.html?q=orbit#examples/en/controls/OrbitControls) needs a reference to the camera and the renderer, you need to pass them as arguments.
+Since [OrbitControls](https://threejs.org/docs/index.html?q=orbit#examples/en/controls/OrbitControls) needs a reference to the camera and the renderer, you need to pass those as arguments.
````
Non-blocking.

> reference to the camera and the renderer, you need to pass

> reference to the camera and the renderer. You need to pass

andretchen0 commented on docs/guide/your-first-scene.md:
```diff
@@ -10,13 +10,13 @@ This guide will help you to create your first Tres scene. 🍩
 ## Setting up the experience Canvas
 
-Before we can create a Scene, we need somewhere to display it. Using plain [ThreeJS](https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene) we would need to create a `canvas` HTML element to mount the `WebglRenderer` and initialize the `scene`
+Before we can create a scene, we need a space to display it. Using plain [ThreeJS](https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene) we would need to create a `canvas` HTML element to mount the `WebglRenderer` and initialize the `scene`
```
(non-blocking)

> Before we can create a ThreeJS `Scene`, we need somewhere to display it. Using plain [ThreeJS](https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene) we would need to create a `<canvas>` HTML element to mount the `WebglRenderer` and initialize the `Scene`.

----

## Explanation

* To me, it's clearer when HTML looks like tags (`<canvas>`).
* Made "Scene" and `scene` consistent. It's a JS class, so typically capitalized so `Scene`.
* Added a missing period at the end.

andretchen0 commented on docs/guide/your-first-scene.md:
````diff
@@ -10,13 +10,13 @@ This guide will help you to create your first Tres scene. 🍩
 ```vue
 <script lang="ts" setup>
-import { TresCanvas } from '@tresjs/core'
+import { TresCanvas } from "@tresjs/core";
````
It'd be best to use the project's linter rules here, I think.

The linter rules (node_modules/@tresjs/eslint-config-base/index.js) are set up to use single quotes and omit optional semicolons.

Same for the other corrected code examples. (I won't add comments as it seems passive-aggressive, lol. 😀 Though if it would be helpful, I can do that.)

colinscz replied:
Thanks, yes absolutely - I think I missed those before I disabled my save actions in VsCode - might be helpful to also export the config settings for the "action on save" in the future. Or disable that "format on save" feature if possible.

andretchen0 commented on docs/guide/your-first-scene.md:
```diff
@@ -109,23 +109,23 @@ Then you can add a [**PerspectiveCamera**](https://threejs.org/docs/index.html?q
 
 ::: warning
-A common issue is that the camera default position is the origin of the scene (0,0,0), TresJS will automatically set the position of your camera to `[3,3,3]` if the prop `position`. If no camera is defined in you scene, a perspective camera is added automatically.`
+A common issue is, that the camera default position is the origin of the scene (0,0,0). TresJS will automatically set the position of your camera to `[3,3,3]` if the prop `position` is not set by you. If no camera is defined in you scene, a perspective camera is added automatically.
```
I'd remove the comma that was added here:

> A common issue is, that

colinscz replied:
removed

andretchen0 reviewed (changes requested):
Thanks for this! Lots of good changes here.

I made some comments. Most are "non-blocking" and are marked.

For those not marked "non-blocking" reply to me if you disagree. Otherwise, if you find the changes acceptable, go ahead and add them to the PR.

(I should say up front that I'm not an editor. My only "qualification" here is being an anglophone working on Tres/Cientos!)

Thanks again!

colinscz commented on docs/examples/lights-shadows.md:
```diff
@@ -62,11 +62,11 @@ import { OrbitControls } from '@tresjs/cientos'
 ## Shadows (explanation)
 
-There are also many types of shadows, for example the "soft shadow" is generated automatially when an object receives more light from one side, but in summary a "ThreeJS default shadow" that is directed towards another surface needs to be cast by a mesh and another mesh needs to receive it. As we see in our example, the `Plane` is receiving a shadow but not casting it. Please note that not all ma ...
+There are many types of different shadows, for example the "soft shadow" is generated automatially when an object receives more light from one side. In conclusion a "ThreeJS default shadow" that is directed towards another surface needs to be cast by a mesh and another mesh needs to receive it. As we see in our example, the `Plane` is receiving a shadow but not casting it. Please note that not al ...
```
I reworked the sentence a bit more, maybe it makes now sense again. Please have a second look at it :)

colinscz commented:
@andretchen0 I addressed all your comments since I like the approach of fixing things that only take a minute right away. 🤙🏽 

Please have a second look at it, I also rephrased the part with the "in summary". But probably one person of the core team should comment on that point anyhow.

colinscz commented on docs/de/guide/nuxt.md:
```diff
@@ -30,7 +30,7 @@ yarn add three @tresjs/nuxt
 
 - 🤓 Automatischer Import von Komponenten und Composables aus dem [TresJS-Ökosystem](https://github.com/orgs/Tresjs/repositories)
 - `TresCanvas` ist nur auf dem Client verfügbar, daher ist es nicht notwendig, `.client` zum Namen der Komponente hinzuzufügen oder `<ClientOnly />` zu verwenden
-- Konfiguriert automatisch den Vue-Compiler, um TresJS-Komponenten zu unterstützen, siehe [warum](http://localhost:5173/de/guide/troubleshooting.html#fehler-beim-auflosen-des-komponenten-trescomponent)?
```
@Tinoooo I fixed this link in the english, here and in the spanish version, where for some reason this anchor was created, after shortening it and removing the extra parts the links still worked as expected

andretchen0 commented on docs/guide/your-first-scene.md:
````diff
@@ -69,7 +69,7 @@ If your scene is not gonna be part of a UI, you can also force the canvas to tak
 ```vue
 <script lang="ts" setup>
-import { TresCanvas } from '@tresjs/core'
+import { TresCanvas } from "@tresjs/core"
````
Single quotes here.

```
import { TresCanvas } from '@tresjs/core'
```

andretchen0 reviewed (changes requested):
Looks good!

There's just one instance of double quotes remaining.

andretchen0 reviewed (approved):
Great! Approved.

colinscz commented:
@alvarosabu from my side this is ready to go - maybe you or @JaimeTorrealba have a quick look at the sentence in the light-shadows.md and if the meaning of it is correct after my update.

JaimeTorrealba reviewed (approved):
Thanks for this, for me is ready

colinscz commented:
> Thanks for this, for me is ready

Great thanks for the feedback @JaimeTorrealba , then I resolve the remaining open threads. I just resolved the merge conflict so it would be ready to merge.
